heimdall-api/src/main/java/br/com/conductor/heimdall/api/resource/AuthResource.java

```diff
@@ -53,9 +55,19 @@
     @Autowired
     private TokenAuthenticationService tokenAuthenticationService;
 
+    @Autowired
+    private ReCaptchaService reCaptchaService;
+
+    @Value(value = "${info.app.captchaKey}")
+    private String secret;
+
+    @ResponseBody
     @ApiOperation(value = "Login Authentication")
     @PostMapping(ConstantsPath.PATH_LOGIN)
     public ResponseEntity<?> login(@RequestBody AccountCredentials accountCredentials, HttpServletResponse response) {
+        if(!accountCredentials.getCaptchaResponse().equals("false")){
```

Comment:
you could  use  Boolean.FALSE.toString() and change the expression, it's not  clean.
try this.
Boolean.TRUE.toString().equals(accountCredentials.getCaptchaResponse())